robot_sf/benchmark/schemas/forecast_batch_schema.py

```
@@ -0,0 +1,212 @@
+        self._load_schema()
+
+    def _load_schema(self) -> None:
+        """Load and validate the schema from file."""
+        if not self.schema_path.exists():
+            raise FileNotFoundError(f"Schema file not found: {self.schema_path}")
+
+        try:
+            with open(self.schema_path, encoding="utf-8") as f:
+                self._schema_data = json.load(f)
+        except json.JSONDecodeError as e:
+            raise ValueError(f"Invalid JSON in schema file {self.schema_path}: {e}")
```

Ruff (0.15.17)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)